bridge/runbook/validator_runbook.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Sui Bridge Validator Runbook
+---
+
+## Prerequisite
+
+Install `sui`, `sui-bridge-cli` binaries:
+```bash
+# install from tip of `main`
+cargo install --locked --git "https://github.com/MystenLabs/sui.git" sui sui-bridge-cli
+# install with a commit sha
+cargo install --locked --git "https://github.com/MystenLabs/sui.git" --rev {SHA} sui sui-bridge-cli
+```
+
+## Committee Registeration
+
+### Prepare for Metadata
+
+The required metadata includes two things:
+* `BridgeAuthorityKey`, a ECDSA key to sign messages. Since this is a hot key that is kept in memory, it’s fine to use the following tool to generate one and write to file.
+* a REST API URL where the bridge node listens to and serves requests. Example: `https://bridge.example-sui-validator.io:443`. Make sure the port is correct.
```

Comment:
```suggestion
* a REST API URL where the bridge node listens to and serves requests. Example: `https://bridge.example-sui-validator.io:443`. Make sure the port is correct and the url does not contain any invalid characters, for example, invalid quotes: `"“bridge.example-sui-validator.io:443”"`
```

(this is a real problem in testnet right now `ERROR sui_proxy::peers: Unable to parse http_rest_url url_str="“https://rpc-testnet.suiscan.xyz:443”"`)